src/components/Bundlr.tsx

```diff
@@ -114,12 +124,18 @@ export function Bundlr({ children }: { children?: React.ReactNode }) {
               </Button>
             </HStack>
             <Text variant="secondary">Bundlr balance: {formattedBalance}</Text>
-            {formattedUploadPrice > ONE_WEI ? (
+            {formattedUploadPrice > ONE_WEI && (
```

Comment:
this is a note from the previous PR -- it looks like `uploadPrice` is a `BigNumber` (converted to a string) -- so this could be instead be:

`BigNumber.from(uploadPrice).gt(BigNumber.from(0)) &&`

That would remove the need for the `ONE_WEI` const.

An even better way to do this would probably be to change the `uploadPrice` type on the embalmerState to: `string | undefined` and then default it to `undefined`. Then we would just need to check the presence of it `uploadPrice &&` -- however, not sure if making this change would break anything.